CHANGELOG.md

```diff
@@ -5,10 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
-## [7.8.0]
+## [8.0.0]
 
-### Added
-- Added `INSAR_MULTI_BURST` job type to `hyp3-test` that can merge adjacent bursts into an SLC and then perform InSAR.
+### Changed
+- `INSAR_ISCE_BURST` now has the ability to process multiple adjacent bursts into one interferogram.
+- `INSAR_ISCE_BURST` now uses `reference` and `secondary` parameters which each accept a list of 1 or more scene names.
```

Comment:
My understanding from this morning's conversation with @forrestfwilliams was that we would be releasing the parameter name change first, and then enabling multi-burst functionality in a subsequent release. I'm open to correction.
```suggestion
- `INSAR_ISCE_BURST` now accepts `reference` and `secondary` parameters which are each a list containing a single granule name. In a future release, `INSAR_ISCE_BURST` will support processing multiple adjacent bursts into one interferogram, and each parameter will be a list of one or more granule names.
```

Reply:
Yep @jtherrmann that's correct. *Technically* the plugin is capable of handling multiple reference/secondary scenes at this point, but we're limiting the number of input scenes to one each for now to disable the functionality. I like this re-write of the changelog.